shuup/core/utils/price_display.py

```diff
@@ -161,6 +161,27 @@ def __call__(self, context, source, include_taxes=None):
         return money(total)
 
 
+def get_priced_children_for_price_range(request, product, quantity, supplier):
+    product_queryset = product.variation_children.visible(
+        shop=request.shop, customer=request.customer
+    ).order_by("shop_products__default_price_value", "pk")
```

Comment:
https://stackoverflow.com/questions/51302345/django-last-and-first-return-the-same-object for this since I guess there is a case when default price value is zero for all children. Then this would ensure that the low and high get different products. Mainly this was needed for the test since it figures out the price from sku.